csrc/mmdeploy/apis/java/mmdeploy/ContextType.java

```diff
@@ -0,0 +1,8 @@
+package mmdeploy;
+
+public enum ContextType {
+    DEVICE,
+    SCHEDULER,
+    MODEL,
+    PROFILER;
```

Comment:
Is `;` legal?

Reply:
> Is `;` legal?

My ContextType.class is successfully built in my localhost.

Reply:
Please follow the convention: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html